### DISCUSSION TITLE
app license is ambiguous, is Status free software or not?

### DISCUSSION THREAD

**Discussion by sneak** [status-im/status-legacy#13778] [General]
According to the `LICENSE.md` in the repository, the app is licensed under Mozilla Public License 2.0.

However, on launching the app, I am presented with a consent screen demanding that I agree to the "Terms of Use", which says, among other things:

> You agree to use the Status app in a way that doesn't harm or violate the rights of Status, Status app users, or others in any way.

I do not agree to this!  In fact, I reserve all rights granted to me under the MPL, including the right to use the software in ways that the copyright owner does not agree with.

If this is indeed a condition of the software license agreement, then Status is not free software / open source, as such usage restrictions make software nonfree.

The Terms of Use should be removed from the mobile app, as the app is software and the only terms of use for the software are the software license agreement.

However, if the Terms of Use govern the use of APIs or services provided by Status Research & Development GmbH in Switzerland, this should be made much more clear, as the messaging around the brand suggests that I am using only public/decentralized/***permissionless*** APIs (such as the Ethereum blockchain) and that it has no reason to phone home.

If it does indeed use **permissioned services** provided by Status Research & Development GmbH, I don't want to be a user.  I want local free software, not an account on some centralized service provider that tells me how I can and can't use the service.  That's a ***permissioned** system, as evidenced by the restrictive demands made of me in the "Terms of Use".

If it does not, then it's only the **software** license that applies to the app, and that's the MPL2, and the whole "Terms of Use" thing is a confusing red herring.

**Comment by du82** [comment]
Status is free, open-source, peer to peer (decentralized) software.

Do whatever you want with the app, (hack it, fork it, have fun) just don't use the Status brand if you release your own version. The brand and app are seperate.

**Comment by sneak** [comment]
If it's free software, why am I asked to agree to a nonfree Terms of Use on first launch?

**Reply by du82** [reply]
That's there to do the bare minimum to comply with Apple and Google's app store policies which require a TOS and Privacy Policy. You can spam all you want (which is technically against the policy) and nothing will happen.

Status does not have an overarching control over users, and can't ban anyone like platforms such as Discord can.
Account suspension does not, and will never exist.

It's peer to peer, so it's impossible by design.

**Reply by sneak** [reply]
> *That's there to do the bare minimum to comply with Apple and Google's app store policies which require a TOS and Privacy Policy. You can spam all you want (which is technically against the policy) and nothing will happen.*

If it has no force, why am I being asked to agree to it to use the software?  If agreeing to it is required, then the client is not open source.

**Reply by easye** [reply]
Non-consent should be supported.

**Reply by sneak** [reply]
Why have a Terms of Use document if they aren't the terms for use?  This is misleading and dishonest.

The only terms of use that govern the use of the client software is the copyright license under which it is released, which is MPL.  Insinuating that there are any other terms of usage for the client software is dishonest.

If the software is used as a client for some network services from Status Research & Development GmbH, then that should be made clear in advance.  I was under the impression that the protocols that the Status client app speaks are **permissionless**, which is to say that they **do not have Terms of Use** and that use is unrestricted.

**Reply by du82** [reply]
> Non-consent should be supported.

I agree. We shouldn't require consent of any kind. If you don't like it, fork the app, remove the checkbox, and compile it yourself. 

The consent button is entirely client side, which is stupid and useless.

**Reply by sneak** [reply]
If you agree, then my forking the project is counterproductive, the change is better done in the upstream.

**Reply by du82** [reply]
I would, but I know my PR would get rejected and labeled as a regression

**Comment by agatalaw** [comment]
If the app is downloaded from the app store and in order for Status to be able to distribute it through an app store, Status must adhere to terms of such app stores and their various and increasingly numerous developer program policies and terms. The app/software that Status distributes in such a way is free to use but not unconditional and free of any restrictions as Status is bound by the contract with the app store as the app store is effectively a distribution agent. This is a good point though and we will look into alternative arrangements for when the app is downloaded through other channels, like our website or f-droid.
